--- table-mode.vim	2014-05-29 23:50:46.000000000 -0700
+++ t.vim	2014-07-21 10:07:37.095579862 -0700
@@ -80,8 +80,8 @@
 command! TableEvalFormulaLine call tablemode#spreadsheet#formula#EvaluateFormulaLine()
 
 " '|' is a special character, we need to map <Bar> instead
-if g:table_mode_separator ==# '|' | let separator_map = '<Bar>' | endif
-execute 'inoremap <silent> <Plug>(table-mode-tableize)' separator_map . '<Esc>:call tablemode#TableizeInsertMode()<CR>a'
+"if g:table_mode_separator ==# '|' | let separator_map = '<Bar>' | endif
+"execute 'inoremap <silent> <Plug>(table-mode-tableize)' separator_map . '<Esc>:call tablemode#TableizeInsertMode()<CR>a'
 
 nnoremap <silent> <Plug>(table-mode-tableize) :Tableize<CR>
 xnoremap <silent> <Plug>(table-mode-tableize) :Tableize<CR>
